TeadsSampleApp/WebViewHelper/TeadsWebViewHelper.swift

```diff
@@ -186,7 +186,7 @@ import WebKit
     }
 
     private func createContainerView(topOffset: CGFloat, bottomOffset: CGFloat) -> UIView {
-        let container = UIView()
+        let container = PassthroughView()
         container.clipsToBounds = true
```

Comment:
maybe set this should do the trick:
`container.isUserInteractionEnabled = false`

Reply:
this will make the ad view not tappable.